_pytest/logging.py

```diff
@@ -101,9 +101,13 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         '--log-date-format',
         dest='log_date_format', default=DEFAULT_LOG_DATE_FORMAT,
         help='log date format as used by the logging module.')
-    parser.addini(
-        'log_cli', default=False, type='bool',
-        help='enable log display during test run (also known as "live logging").')
+    add_option_ini(
+        '--log-stream',
```

Comment:
Why did you invent a new name? Why not --log-cli?

Reply:
Seems weird, `--log-cli`, and `--log-stream` just seemed more explicative. I have no issues making it `--log-cli` though...